### ISSUE TITLE
Add Drag n' Drop support for Sprite3D

### ISSUE THREAD

**Issue by julhe** [godotengine/godot-proposals#5204]
### Describe the project you are working on

3D Game with lots of 2D sprites.

### Describe the problem or limitation you are having in your project

It's cumberstone to create a new Sprite3D each time I want to add a sprite to the 3D scene.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Allow for Drag n' Drop, in the same fashion as for 2D.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

N/A

### If this enhancement will not be used often, can it be worked around with a few lines of script?

N/A

### Is there a reason why this should be core and not an add-on in the asset library?

N/A

**Comment by Mickeon** [comment]
Drag 'n' dropping from file-system to window, in general, hasn't quite worked for me in Godot 4 for a while and I can't understand why...

**Comment by Mickeon** [comment]
One problem with this is that dragging a Texture2D/Material to the viewport allows overriding a Mesh's texture, so the proposed behaviour should be probably accessible with a shortcut.

**Comment by julhe** [comment]
@Mickeon Yeah, I guess it conflicts with the behaviour. Maybe it's possible to change the default behaviour with holding the alt key, similiar to how it is done in 2D?

**Comment by Mickeon** [comment]
One other complication I'm encountering is that scale is a big issue, yeah. The texture may be small, which results into the Sprite3D looking small. A Popup when holding the Alt Key looks to be necessary at this point.

If this... feature is particularly useful, even. Typically, in a 3D environment one may want more granularity: splitting Nodes into scenes, each of them is a Sprite3D, copying and pasting that Scene, instead of the raw Sprite3D...
